kubernetes/README.md

```diff
@@ -25,7 +25,37 @@ Follow [this](https://docs.gitlab.com/ee/user/project/clusters/add_remove_cluste
 ```
 kubectl -n polkassembly create secret generic regcred --from-file=.dockerconfigjson=$HOME/.docker/config.json --type=kubernetes.io/dockerconfigjson
 kubectl -n polkassembly get secrets
-```  
+```
 to create the *secret* containing the required credentials in k8s.
 * make the credentials available from within gitlab-ci (required to push images to the registry).
 `cat $HOME/.docker/config.json | tr -d '\n'`. cut&paste resulting String into gitlab-ui -> Settings -> CI/CD -> Variables.
+
+4. Manual Steps needed after deployment to run polkassembly in production:
+
+- Create a bot user and set its id/credentials in auth server and chain db watcher
+  - create a user named Bot User through signup
+  - set botuser username in gitlab ci variable PROPOSAL_BOT_USERNAME. Needed as env var in chain db watcher.
+  - set botuser password in gitlab ci variable PROPOSAL_BOT_PASSWORD. Needed as env var in chain db watcher.
+  - set botuser user id in gitlab ci variable PROPOSAL_BOT_USER_ID. Needed as env var in auth server and chain db watcher.
+- Run migrations in hasura
+  We are using remote join branch of hasura where automatic migrations are not running. We need to run them maually.
+  The steps are:
+  - Clone the hasura cli and copy it in a directory in your PATH.
+
+    ```bash
+    git clone https://github.com/hasura/preview-remote-joins.git
+    mv cli/cli-hasura-<arch> /usr/local/bin/hasura-dev
+    chmod +x /usr/local/bin/hasura-dev
+    ```
+  - Port forward local 8080 port to hasura service
+    - kubectl get service --namespace=polkassembly
+    - kubectl port-forward svc/<hasura-service> 8080:8080 --namespace=polkassembly
+  - run `hasura-dev migrate apply --admin-secret HASURA_GRAPHQL_ADMIN_SECRET`
+- Reload metadata
+ - This is needed after each deployment of auth server/hasura/node watcher
+ - Expose hasura with a load balancer in k8s
+ - goto ip:8080/console
```

Comment:
this will be `localhost:8080/console` right?
```suggestion
 - goto localhost:8080/console
```

Reply:
no this need to be loadbalancer ip